tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,188 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "Domain/Direction.hpp"
+#include "Domain/Orientation.hpp"
+#include "Utilities/StdHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+void test_1d() {
+  // Test constructors:
+  Orientation<1> custom1(
+      std::array<Direction<1>, 1>{{Direction<1>::upper_xi()}});
+  CHECK(custom1.is_aligned() == true);
+  Orientation<1> custom2(
+      std::array<Direction<1>, 1>{{Direction<1>::lower_xi()}});
+  CHECK(custom2.is_aligned() == false);
+
+  // Test if Orientation can encode a 1D parallel/ antiparallel.
+  std::array<Direction<1>, 1> block1_directions{{Direction<1>::upper_xi()}};
+  std::array<Direction<1>, 1> block2_directions{{Direction<1>::lower_xi()}};
+  Orientation<1> parallel_orientation(block1_directions, block1_directions);
+  Orientation<1> antiparallel_orientation(block1_directions, block2_directions);
+  std::array<SegmentId, 1> segment_ids{{SegmentId(2, 1)}};
+  std::array<SegmentId, 1> expected_antiparallel_segment_ids{{SegmentId(2, 2)}};
+  CHECK(parallel_orientation.mapped(segment_ids) == segment_ids);
+  CHECK(antiparallel_orientation.mapped(segment_ids) ==
+        expected_antiparallel_segment_ids);
+  CHECK(get_output(parallel_orientation) == "(+0)");
+  CHECK(get_output(antiparallel_orientation) == "(-0)");
+  CHECK(custom1 != custom2);
+  CHECK(custom1 == parallel_orientation);
+}
```

Comment:
also for each dimension, call test_copy_semantics and test_move_semantics (defined in tests/Unit/TestHelpers.hpp)